py/makeqstrdata.py

```diff
@@ -9,17 +9,54 @@
 import re
 import sys
 
-# Python 2/3 compatibility:
+# Python 2/3/micro compatibility:
 #   - iterating through bytes is different
-#   - codepoint2name lives in a different module
-import platform
-
-if platform.python_version_tuple()[0] == "2":
+#   - codepoint2name from html.entities is hard-coded
+if sys.version_info[0] == 2:
     bytes_cons = lambda val, enc=None: bytearray(val)
-    from htmlentitydefs import codepoint2name
-elif platform.python_version_tuple()[0] == "3":
+elif sys.version_info[0] == 3:  # Also handles MicroPython
     bytes_cons = bytes
-    from html.entities import codepoint2name
+
+# fmt: off
+codepoint2name = {
```

Comment:
Is this copied verbatim from the Python source?  Is there any licensing/copyright issue doing that?

Is it worth instead making a `micropython-lib` module for `html.entities` that has this dict there?

(In general I'm a fan of keeping things self contained, so having this dict verbatim here, but in this case maybe it's better to make a library for it.)